code/modules/mining/mine_items.dm

```diff
@@ -284,24 +288,20 @@
 	var/area/survivalpod/L = new /area/survivalpod
 
 	var/turf/threshhold = locate(start_turf.x, start_turf.y-2, start_turf.z)
-	threshhold.ChangeTurf(/turf/open/floor/plasteel/pod)
-	threshhold.blocks_air = 1 //So the air doesn't leak out
-	threshhold.initial_gas_mix = "o2=21;n2=82;TEMP=293.15"
+	threshhold.ChangeTurf(/turf/open/floor/pod)
+	var/turf/open/floor/pod/doorturf = threshhold
+	doorturf.blocks_air = 1 //So the air doesn't leak out
+	doorturf.initial_gas_mix = "o2=21;n2=82;TEMP=293.15"
+	doorturf.air.copy_from_turf(doorturf)
```

Comment:
replace these lines with
`doorturf.air.parse_gas_string("o2=21;n2=82;TEMP=293.15")`
it's my fault for being dumb, not yours